src/Compiler/Microsoft.AspNetCore.Mvc.Razor.Extensions.Version2_X/test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/RazorPagesWithoutModel_Runtime.ir.txt

```diff
@@ -26,6 +26,8 @@
                     LazyIntermediateToken - (7:1,0 [2] RazorPagesWithoutModel.cshtml) - Html - \n
                 CSharpCode - 
                     IntermediateToken -  - CSharp - EndContext();
+                CSharpCode - (79:3,42 [2] RazorPagesWithoutModel.cshtml)
```

Comment:
This is the newline the user wrote that is no longer part of the using directive